lib/src/models/candle/chart_candle_data_item_value.dart

```diff
@@ -1,8 +1,13 @@
 part of 'chart_candle_layer.dart';
 
+/// Provides data item of candle.
 class ChartCandleDataItemValue {
```

Comment:
This is something different ChartCandleDataItem, should have more detailed description